be/src/connector_sink/file_chunk_sink.cpp

```diff
@@ -93,11 +94,11 @@ std::function<void(const formats::FileWriter::CommitResult& result)> FileChunkSi
     };
 }
 
-StatusOr<std::unique_ptr<ConnectorChunkSink>> FileChunkSinkProvider::create_chunk_sink(
+std::unique_ptr<ConnectorChunkSink> FileChunkSinkProvider::create_chunk_sink(
         std::shared_ptr<ConnectorChunkSinkContext> context, int32_t driver_id) {
     auto ctx = std::dynamic_pointer_cast<FileChunkSinkContext>(context);
     auto runtime_state = ctx->fragment_context->runtime_state();
-    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateUniqueFromString(ctx->path, FSOptions(&ctx->cloud_conf)));
+    auto fs = FileSystem::CreateUniqueFromString(ctx->path, FSOptions(&ctx->cloud_conf)).value();
```

Comment:
is it a blocking operation executed in pipeline executor thread? if so, the blocking operations must be changed to  non-blocking style.

Reply:
non-blocking. `CreateUniqueFromString` only invokes ctor of relevant file system.